assembl/static2/js/app/components/debate/common/postActions.jsx

```diff
@@ -61,13 +61,14 @@ class PostActions extends React.Component {
     const modalTitle = <Translate value="debate.sharePost" />;
     const useSocial = debateData.useSocialMedia;
     let overflowMenu = null;
+    const tooltipPlacement = this.state.screenWidth >= MEDIUM_SCREEN_WIDTH ? 'left' : 'top';
```

Comment:
Ahh, this may have strange consequences for the stats hovering on all sentiments. On a smaller screen, you're going to show a big stats box on "top" of what you were reading above. Don't think it's a dev problem at all, but maybe a UX one.

Reply:
I didn't notice any shocking problem of this kind when testing. But this is not new anyway, the tooltip was always on top for small screen